src/term.js

```diff
@@ -2019,7 +2020,11 @@ Terminal.prototype.write = function(data) {
           //   dow) (DECSTBM).
           // CSI ? Pm r
           case 'r':
-            this.setScrollRegion(this.params);
+            if (!this.disableScrollingRegion) {
+              this.setScrollRegion(this.params);
+            } else {
+              console.log("skipping setScrollRegion");
```

Comment:
I don't think logging this is necessary.

Reply:
removed